other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,181 @@
+# Launching a Site on Altis
+
+The process for launching a site on Altis varies depending on the type of project. Nonetheless, there are common steps that need to be done no matter the context and this guide will walk you through what needs to be done in preparation for your site launch.
+
+## Planning
+
+Create a launch issue in the project repository. This issue should include:
```

Comment:
We should note advice on setting require login to true for the production environment until after the domains and everything have been setup and migrated to, then as a last step, require login should be removed, committed and deployed